pom.xml

```diff
@@ -26,9 +26,19 @@
     </properties>
     <dependencies>
         <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-            <version>1.2.17</version>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-api</artifactId>
+            <version>2.13.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+            <version>2.13.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-slf4j-impl</artifactId>
```

Comment:
Note: we include the `log4j-slf4j-impl` jar from log4j2 in order to properly bridge logs from kafka-clients which log through slf4j to log4j2. Likewise `slf4j-log4j12` is excluded from `kafka_2.12` below in order to avoid overlapping shaded jars of incompatible log4j verisons